src/flavors/williams/config.yml

```diff
@@ -279,6 +279,108 @@ left_sidebar:
               icon: /static/css/images/flood-zones.svg
               title: _(Flood Zones)
 
+    - id: legend
```

Comment:
It doesn't look like a legend is rendered for this `williams` flavor. If the legend is not supposed to render, then we shouldn't include this information in the config.

Reply:
I'm not sure why no legend appears. All the VV flavors render legends when I run this branch:

<img width="1268" alt="screen shot 2018-09-09 at 6 57 42 am" src="https://user-images.githubusercontent.com/9222338/45265312-b9cb3400-b3fd-11e8-84c2-72273ee2dd92.png">
<img width="1270" alt="screen shot 2018-09-09 at 6 57 53 am" src="https://user-images.githubusercontent.com/9222338/45265314-bcc62480-b3fd-11e8-9d9e-2770b4c60f59.png">

Reply:
ah, gotcha. Thanks for explaining that :+1: 

I thought the MapLayerPanel was supposed to be the MapLegendPanel.